openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1326,7 +1326,11 @@ To sender-constrain Access Tokens, see the recommendations in Section 4.10.1 in
 
 # Privacy Considerations
 
-TBD
+## Data Minimization in the Issued Credential
+
```

Comment:
Probably not directly related to this specific PR, however the server metadata for `jwt_vc_*` formats does include `credentialSubject`. I.e. `credentialSubject` is not used only on requests from the wallet to the server but also on information _advertised_ by the server. Therefore, shouldn't it also be optionally included in the offer, namely when the offer does not match any `credentials_supported` entry in the metadata?

Reply:
Question related to https://github.com/openid/OpenID4VCI/issues/82

Reply:
yes, I think this PR might become redundant based on #77 too